### PR TITLE
Reflects on one year of the project

### PR DESCRIPTION
**Pull request by andreakb**
I thought it might be a good idea to post a blog reflecting on one year of this project. I wrote up a quick draft as an idea so please take a look and edit and add stuff (links probs?) etc.  Thanks!

### CODE REVIEW

File: _posts/2019-01-07-One-year-in.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+<add more examples>
+
+# Looking forward 
+
+When we originally started this project, we also wanted it to be a space where people could share their own positive and negative experiences in the field, anonymously if they chose. And, while we haven't done that yet, it is still something we want to be a platform for. If anyone has suggestions about how to create a safe, anonymous 'tip line' of sorts to facilitate this, let us know!
+
+While Code of Conducts are a fine ethical barometer for the digital preservation community, there are still lots of systemic issues that need to be addressed, assessed, and corrected. One of these is the need for hiring institutions value our labor monetarily. While it is great that loads of people and institutions are coming around to digital preservation, and there are more professional opportunities for us to do our work, and while we love what we do, we need to be paid accordingly. 
+
+<something about open source projects being done on a voluntary basis here maybe>?
```
Comment (ross-spencer) [inline _posts/2019-01-07-One-year-in.md]: Maybe something like, 

> As more and more folk identify technology, not just as a time-saver, but as a necessity to their work, they need to be supported in learning, and developing. But information technology reduces the barriers to learning enabling people to do tutorials and courses for free. Programming languages make it easy to engage with commodity hardware and so everything can be done at home. How do we make sure there are opportunities. How do we make sure that folks are supported and rewarded for their endeavors, how do we make sure the output of this effort is not perpetually done on a voluntary basis and brought back to organisations, organisations push and not just pull. 

>  And that should work the same for projects outside of an organisation's remit. Open source tools, those developed, not just by other organisations, but those developed by those, at home at the weekend, trying to solve another problem for themselves in their work, can be supported better too. Mentions and acknowledgements in papers, blogging, and conferences, can go a long way. Reporting bugs or issues lets folk know the tool is being used. Sharing the results of in-house evaluations in a generous way, or equally, the results of in-house customization. There are many ways, and the smallest thing to one person, or an organisation, might be the biggest thing to an individual out there.

**NB.** There's something in my first paragraph that might speak to the idea of the analyst-developer @ablwr has been meditating on recently, she might have a few ideas about links?

Comment (ablwr) [reply _posts/2019-01-07-One-year-in.md]: Appreciate this!! Will be thinking on it...

**Comment by ablwr** [comment]
Thank you so much @andreakb !

File: _posts/2019-01-07-One-year-in.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Some incremental changes were made, and we can celebrate those changes while looking forward to what's next and what further work can be done. As in digital preservation, the work is in the maintenance, not just the creation. 
+
+Some highlights include:
+
+* [iPres 2018 adopted the first iPres code of conduct and response framework](https://ipres2018.org/code-of-conduct)
+* [The Digital Preservation Coalition created an Inclusion and Diversity policy](https://www.dpconline.org/docs/miscellaneous/about/1807-dpc-inclusion-and-diversity-policy/file)
+* The 2018 Digital Library Foundation Forum's Call for Proposals stated that: ["“Manels”—all-male-identifying panels—will not be accepted into the conference, and proposals involving a wide diversity of speakers will be looked on favorably by the committee."](https://forum2018.diglib.org/call-for-proposals/)
+
+<add more examples>
```
[the remark this answers is not in the document]
Comment (andreakb) [reply _posts/2019-01-07-One-year-in.md]: Good idea! I'll post an issue, thanks!

File: _posts/2019-01-07-One-year-in.md (hunk 1 above)
Comment (andreakb) [reply _posts/2019-01-07-One-year-in.md]: Ooooh, yes, good writing & good idea! @ablwr talking about this on twitter around No Time to Wait and beyond made me think about this, perfect! Thank you!

Comment (ablwr) [reply _posts/2019-01-07-One-year-in.md]: Wish I had something more concrete or could drum up something real quick right now, but afraid I have to go back to doing all the things that y'all are writing about, hahaha!! ;) If the intent for 2019 is to blog more, I could shrink up an unfinished manifesto-type academic-type writing on this topic, which hopefully my co-author and I can form into something bigger this year (and get accepted! we got two rejections so we put it on the back-burner for a bit to focus on other projects).

Comment (andreakb) [reply _posts/2019-01-07-One-year-in.md]: No worries! And yeah, that sounds awesome, whenever you're ready!

Comment (andreakb) [reply _posts/2019-01-07-One-year-in.md]: Hey @ross-spencer, I'm going to add your paragraphs as is, if that's okay with you, and post this bad bear. Thanks so much for your help!

**Comment by kfrn** [comment]
Late to the party, but @andreakb, this looks awesome! Thanks heaps for writing this up.

Re: pay, I actually thought that digipres staff were paid pretty well, is that not the case?

**Comment by kfrn** [comment]
Late to the party, but @andreakb, this looks awesome! Thanks heaps for writing this up.

Re: pay, I actually thought that digipres staff were paid pretty well, is that not the case?

**Comment by ablwr** [comment]
I'd say they are paid pretty well in relation to other people in the GLAM field, but I think that pay is often unequal across gender (and other) lines?

**Comment by ablwr** [comment]
Also I've seen some brutally underpaid jobs coming especially out of the U.K. this year. I do the exchange rate and think, how can anyone live on that in one of the most expensive cities in the world (London, in this case)?

**Comment by kfrn** [comment]
Yeah, re: London/UK, the GLAM sector there seems to be very poorly paid, even specialist roles aren't much. When I met Somaya, this came up in conversation - I think she said that wages basically never recovered from the GFC.

I think I've mentioned this before, but I like the idea of an anonymous salary spreadsheet, though maybe it's too small a field to pull that off!

**Comment by ablwr** [comment]
@kfrn y'know I just saw one of those going around for Library workers, I saw it on Twitter and then it was lost to the waves. But I think one that is based around our field specifically would have a lot of value. I'd like to know location and gender as well. A good project goal for DPiP?

I'm pronouncing that like "duh PIHHPPP"

**Comment by ross-spencer** [comment]
In case folks are interested. I started a repo for salaries a short while back - I plan periodically copy the DPC jobs RSS feed across, and need to write a script to convert it to CSV, but I hope it will illuminate some of the geographical gaps. Maybe some of the institutional gaps as well. 

Repo here: https://github.com/digital-preservation-is-people/digipres-salaries

**Comment by kfrn** [comment]
Oh cool @ross-spencer! It would be great to have a bunch of that data to analyse.

**Comment by andreakb** [comment]
I think putting together some kind of anonymous salary survey thing is a great project goal in conjunction with @ross-spencer 's WIP. I'll make an issue.  Thanks, all!

**Pull request merged**